alembic/README

```diff
@@ -0,0 +1 @@
+Generic single-database configuration.
```

Comment:
This appears to be a default README file - either this needs expanding to explain what this directory actually contains, or just remove this README as redundant.